data/human/transport missions.txt

```diff
@@ -3209,3 +3209,73 @@ mission "Earth Retirement"
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+mission "Human to Hai Space"
+	minor
+	description "Bring a human passenger to <destination> in Hai space."
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "tourism" "urban"
+	destination "Greenwater"
+	passengers 1
+	to offer
+		random < 10
+	on offer
+		conversation
+			`A young person approaches you carrying luggage. "You look like a knowledgeable captain," he says. "Do you know of a place I can go to visit some... intelligent rodents?"`
+			branch "hai contact"
+				has "First Contact: Hai: offered"
+			
+			choice
+				`	"What are you talking about?"`
+					goto "confused"
+				`	"What, like in a zoo?"`
+					goto "confused"
+				`	"Are you high?"`
+			`	"No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
+				decline
+			label "confused"
+			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
+				decline
+			
+			label "hai contact"
+			choice
+				`	"You mean the Hai? You'll have to go up north for that."`
+					goto "squirrel transport"
+				`	"Ah, I think I know of some large squirrels to the north of here."`
+					goto "squirrel transport"
+				`	"What are you talking about?"`
+					goto "confused"
+			label "squirrel transport"
+			`	"If that's the case, could you bring me to <destination>? I'll give you <payment>."`
+			choice
+				`	"Alright, I'll take you there."`
+					accept
+				`	"Sorry, I'm too busy right now."`
+					decline
+				`	"Why would you want to go all the way out to <planet>?"`
+			`	"I have family there," he says. "I came out here to do some traveling and see a bit of human space, but now I have to get back home. You see, I was born on <planet>, and this is my first time out of the wormhole."`
+			choice
+				`	"Really? How did you like human space?"`
+					goto "travel"
+				`	"You've never been to human space before?"`
+					goto "space"
+			label "travel"
+				`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`
```

Comment:
Currently the "travel" part continues into the "space" part. I think it should go straight to the next choice.

Reply:
Well, now it's the other way around. In game, you'll get: 
"So anyways, would you be willing to take me to <destination>?" 
and then, instead of letting you answer, he'll continue with the "travel" part.

Reply:
Unrelated to my previous comments: there is an extra tab.
```suggestion
			`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`
```